packages/flutter/lib/src/painting/text_painter.dart

```diff
@@ -235,15 +236,67 @@ class TextPainter {
   double get preferredLineHeight {
     if (_layoutTemplate == null) {
       final ui.ParagraphBuilder builder = ui.ParagraphBuilder(
-        _createParagraphStyle(TextDirection.rtl),
-      ); // direction doesn't matter, text is just a space
+        _createParagraphStyle(TextDirection.ltr),
+      ); // direction doesn't matter, text is just a space, ltr is more common.
```

Comment:
Please see https://master-docs-flutter-io.firebaseapp.com/flutter/dart-ui/TextDirection-class.html
We never use LTR as the default (with one exception in the material library)